mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -11,6 +11,30 @@ class _BaseBuiltInScorer(BuiltInScorer):
     inherit from this class.
     """
 
+    def __call__(self, *args, **kwargs):
```

Comment:
```suggestion
    def __call__(self, *, **kwargs):
```
Let's disable passing positional arguments for simplification.